### PR TITLE
feat: preview a visualforce page with open command

### PR DESCRIPTION
**Pull request by fodilo**
### What does this PR do?
so it allows the preview of visualforce page directly in the browser using source:open command with .page files
### What issues does this PR fix or reference?
I've created an issue about this feature [https://github.com/forcedotcom/cli/issues/1386](url)

### CODE REVIEW

**Comment by fodilo** [comment]
I've signed the cla.

File: messages/open.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+  "description": "edit a Lightning Page with Lightning App Builder\nOpens the specified Lightning Page in Lightning App Builder. Lightning Page files have the suffix .flexipage-meta.xml, and are stored in the flexipages directory. \nIf you specify a visualforce page (with a .page suffix) this command let you preview your visualforce page. If you specify a different type of file, this command opens your org’s home page.\n\nThe file opens in your default browser.\nIf no browser-based editor is available for the selected file, this command opens your org's home page.\nTo generate a URL for the browser-based editor but not open the editor, use --urlonly.",
```
Comment (jshackell-sfdc) [inline messages/open.json:2]:
```suggestion
  "description": "edit a Lightning Page with Lightning App Builder\nOpens the specified Lightning Page in Lightning App Builder. Lightning Page files have the suffix .flexipage-meta.xml, and are stored in the flexipages directory. \nIf you specify a Visualforce page, which has a .page suffix, the page opens in your browser so you can preview it. If you specify a different type of file, this command opens your org’s home page.\n\nThe file opens in your default browser.\nIf no browser-based editor is available for the selected file, this command opens your org's home page.\nTo generate a URL for the browser-based editor but not open the editor, use --urlonly.",
```

**Review by jshackell-sfdc — DISMISSED** [review]
See my edit of the command description.

**Comment by mshanemc** [comment]
Thanks for submitting! 🏆  

I made some edits, so I closed this PR and started https://github.com/salesforcecli/plugin-source/pull/419 from your commit.

This also lets our integration tests run, which they won't do on a fork.

**Comment by fodilo** [comment]
> Thanks for submitting! 🏆
> 
> I made some edits, so I closed this PR and started #419 from your commit.
> 
> This also lets our integration tests run, which they won't do on a fork.

Welcome thanks for the review

**Pull request closed without merging**